pages/result.html

```diff
@@ -6,6 +6,19 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="../assets/css/result.css">
     <title>Result Page</title>
+    <meta property="og:title" content="Quiz Result">
+    <meta property="og:description" content="See what your result is!">
+    <meta property="og:image" content="https://factory-kim.github.io/res/1_result_kor_pleasure_and_joy_seeker.jpg">
+    <meta property="og:url" content="https://factory-kim.github.io/pages/start">
```

Comment:
https://blog.bitsrc.io/how-to-generate-previews-for-your-web-page-on-social-media-f867bfd26859

Reply:
Need to update the image